examples/cross_build/toolchain_packages/toolchain_macos_linux_cross/README.md

```diff
@@ -0,0 +1,53 @@
+# MacOs to Linux cross-compilation example
```

Comment:
I think that this title should reflect what this is all about that is creating a toolchain package for cross-building from macOS to Linux